Formula/cadence-workflow.rb

```diff
@@ -0,0 +1,33 @@
+class CadenceWorkflow < Formula
+  desc "Cadence Workflow is a distributed, scalable, durable, and highly available orchestration engine to execute asynchronous long-running business logic in a scalable and resilient way"
+  homepage "https://cadenceworkflow.io/"
+  url "https://github.com/uber/cadence.git",
+      tag:      "v0.18.1",
+      revision: "b6dfe37ebfec7742b1470be7129f07518e8bec98"
```

Comment:
Do we need a `git` checkout, or can we use a tarball?

Reply:
Good point. Looks like we could do that. Thanks